docs/source/users/index.md

```diff
@@ -979,6 +979,19 @@ configuration.
 jupyter lab --AiExtension.allowed_providers=openai --AiExtension.allowed_providers=ai21
 ```
 
+### Chat memory size
+
+This configuration allows for setting the number of chat exchanges the model
+uses as context when generating a response.
+
+One chat exchange corresponds to a user query message and its AI response, which counts as two messages.
+k denotes one chat exchange, i.e., two messages.
+The default value of k is 2, which corresponds to 4 messages.
+
```

Comment:
Add a sentence here: "For example, if we want the default memory to be 4 exchanges, then use the following command line invocation when starting Jupyter Lab:"